webview-ui/src/Periscope/NodeActions.tsx

```diff
@@ -0,0 +1,37 @@
+import { VSCodeButton, VSCodeLink } from "@vscode/webview-ui-toolkit/react";
+import styles from "./Periscope.module.css";
+
+export interface NodeActionsProps {
+    runId: string
+    nodeName: string
+    containerUrl: string
+    shareableSas: string
+    isUploaded: boolean
+}
+
+export function NodeActions(props: NodeActionsProps) {
+    const shareableLink = `${props.containerUrl}/${props.runId}/${props.nodeName}/${props.nodeName}.zip${props.shareableSas}`;
+
+    function copyShareLink(e: React.MouseEvent) {
+        e.stopPropagation();
+        navigator.clipboard.writeText(shareableLink);
+    }
+
+    return (
+        <>
+            <VSCodeButton onClick={copyShareLink}>
```

Comment:
To reduce clutter here, I removed the `Get node logs` button, and instead you just select a row (which corresponds to a node) by clicking on it. Then information about the selected row (which now is just logs, but in theory it could be anything) is displayed below. To me that's intuitive, and gives us more scope to develop UI specific to each node without cluttering the table, but I'm open to further discussion.

Reply:
@rzhang628 - feel free to weigh in here.

Reply:
I like the new UI, much more convenient for a lazy me lol. I am good with approving this one :)